packages/api-client/src/conversation/ConversationAPI.ts

```diff
@@ -459,13 +467,33 @@ export class ConversationAPI {
     const config: AxiosRequestConfig = {
       data: messageTimerData,
       method: 'put',
-      url: `${ConversationAPI.URL.CONVERSATIONS}/${conversationId}/message-timer`,
+      url: `${ConversationAPI.URL.CONVERSATIONS}/${conversationId}/${ConversationAPI.URL.MESSAGE_TIMER}`,
     };
 
     const response = await this.client.sendJSON<ConversationMessageTimerUpdateEvent>(config);
     return response.data;
   }
 
+  /**
+   * Update the receipt mode for a conversation.
+   * @param conversationId The conversation ID
+   * @param receiptModeData The new receipt mode
```

Comment:
Why not calling it just "receiptMode"?

Reply:
Because the interface is called `ConversationReceiptModeUpdateData` and includes `receipt_mode`. I also think it could maybe be a bit too much but I wanted to streamline it with other functions which also take an object. Should I update the text / the parameter name?

Reply:
I would argue that this is signaling that it is an object and not just a value. Also a common pattern we already have in the api-client.